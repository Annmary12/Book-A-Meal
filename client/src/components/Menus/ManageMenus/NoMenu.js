import React from 'react';
import menuImage from '../../../../assets/img/menu.svg';
import styles from './no-menu.scss';

const NoMenu = () => (
  <div className={styles.container}>
    <img
      src={menuImage}
      width="150"
      alt="no menu"
    />
    <p className={styles.text}>Hungry customers are waiting. Set a menu!</p>
  </div>
);

export default NoMenu;

