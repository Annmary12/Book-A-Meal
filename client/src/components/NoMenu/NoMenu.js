import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';
import steamingFood from '../../../assets/img/steaming.svg';
import styles from './no-menu.scss';

const NoMenu = ({ loggedIn }) => (
  <div className={styles.noMenu}>
    <img src={steamingFood} alt="No meal" /><br />
    No menu has been set for the day.<br />
    Our caterers are preparing something amazing!.<br /><br />
    {
      !loggedIn && <div> <Link to="/signup">Sign up</Link> to be notified when menus are set</div>
    }
  </div>
);

NoMenu.propTypes = {
  loggedIn: PropTypes.bool.isRequired
};

export default NoMenu;
