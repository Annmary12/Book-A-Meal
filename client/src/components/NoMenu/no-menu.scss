.noMenu {
  line-height: 1.5;
  text-align: center;
}
