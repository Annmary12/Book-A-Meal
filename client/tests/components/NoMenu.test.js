import React from 'react';
import NoMenu from '../../src/components/Menus/ManageMenus/NoMenu';

describe('NoMenu component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<NoMenu />);

    expect(wrapper).toMatchSnapshot();
  })
})