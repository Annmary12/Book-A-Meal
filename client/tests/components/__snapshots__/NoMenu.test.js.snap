// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMenu component should match snapshot 1`] = `
<div
  className="container"
>
  <img
    alt="no menu"
    src="test-file-stub"
    width="150"
  />
  <p
    className="text"
  >
    Hungry customers are waiting. Set a menu!
  </p>
</div>
`;
